### PR TITLE
Mark's feedback related updates

### PR DESCRIPTION
**Pull request by sandeepsnairms**
1. Updates to Bicep folder based on Mark's feedback.

### CODE REVIEW

File: Bicep/README.md [hunk 1]
```diff
@@ -126,32 +137,62 @@ spec:
...
 
 ```azurecli
-kubectl apply -f secretproviderclass.yaml
+kubectl apply -f secretproviderclass.yml
 ```
 
 **9. Push the container image to Azure Container Registry**
 
-Using Visual Studio build the application source code available in the Application folder, [publish the container image to the ACR](https://docs.microsoft.com/en-us/visualstudio/containers/hosting-web-apps-in-docker?view=vs-2022).
+The application can be built and pushed to ACR using Visual Studio or VS Code, use one of the two methods given below
```
Comment (sajeetharan) [inline Bicep/README.md:153]: use any would be more appropriate

**Review by sajeetharan — COMMENTED** [review]
LGTM

**Review by sajeetharan — APPROVED** [review]
LGTM

**Comment by markjbrown** [comment]
All looks good. Feel free to merge.

**Pull request merged**